### ISSUE TITLE
ChromeDriver has a bug on <object> and <embedded>

### ISSUE THREAD

**Issue by acdcjunior** [seleniumQuery/seleniumQuery#93]
The error thrown is:

```
java.lang.ClassCastException: com.google.common.collect.Maps$TransformedEntriesMap cannot be cast to org.openqa.selenium.WebElement
```

This bug is tracked at https://bugs.chromium.org/p/chromedriver/issues/detail?id=1276
Should be fixed soon.

**Comment by acdcjunior** [comment]
Seems fixed in 2.21. Gotta confirm.

**Comment by acdcjunior** [comment]
Even 2.22 is out. We gotta check this...
